### PR TITLE
Adds support for Linkind PIR motion sensor

### PR DESCRIPTION
**Pull request by jbrechtel**
I inferred which `zonestatus` values meant what based on what I observed from the sensor. I didn't see any other converters which had similar conversions already.

I imagine there are more nuanced data coming from this sensor that I haven't seen yet, though.

### CODE REVIEW

**Comment by jbrechtel** [comment]
There are a few commits fixing tests and PRs, should I squash them?

File: devices.js [hunk 1]
```diff
@@ -9249,6 +9249,15 @@ const devices = [
         description: 'Zigbee A60 filament bulb 6.3W',
         extend: generic.light_onoff_brightness,
     },
+    {
+        zigbeeModel: ['ZB-MotionSensor-D0003'],
+        model: 'ZS1100400-IN-V1A02',
+        vendor: 'Linkind',
+        description: 'PIR Motion Sensor, Wireless Motion Detector',
+        supports: 'occupancy',
+        fromZigbee: [fz.linkind_occupancy],
```
Comment (Koenkk) [inline devices.js:9258]: Please use https://github.com/Koenkk/zigbee-herdsman-converters/blob/915709676fd27220920bc504d668339d6cb10bc4/converters/fromZigbee.js#L682 (fz.ias_occupancy_alarm_1)

Comment (jbrechtel) [reply devices.js]: :+1:  Thanks, I didn't realize that was what I wanted. Done.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**